charts/grafana/templates/_helpers.tpl

```diff
@@ -100,3 +100,16 @@ Selector labels ImageRenderer
 app.kubernetes.io/name: {{ include "grafana.name" . }}-image-renderer
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
+
+{{/*
+Looks if there's an existing secret and reuse its password. If not it generates
+new password and use it.
+*/}}
+{{- define "grafana.password" -}}
+{{- $secret := (lookup "v1" "Secret" (include "grafana.namespace" .) (include "grafana.fullname" .) ) -}}
```

Comment:
==> Linting charts/grafana
Error:  templates/: parse error at (grafana/templates/_helpers.tpl:109): function "lookup" not defined

Reply:
Lookup func is introduced on Helm v3